src/View/Components/Card.php

```diff
@@ -8,17 +8,23 @@
 
 class Card extends Component
 {
+    protected string $textColor = 'text-secondary-700 dark:text-secondary-400';
+
+    protected string $borderColor = 'border-secondary-200 dark:border-secondary-600';
+
     public function __construct(
         public ?string $title = null,
         public ?string $padding = null,
         public ?string $shadow = null,
         public ?string $rounded = null,
         public ?string $color = null,
+        public ?bool $undivided = null,
```

Comment:
Can it be borderless?
```suggestion
        public ?bool $borderless = null,
```